sum-an-array/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Sum Array</title>
+</head>
+
+<body>
+    <script>
+        var Array = 0;
```

Comment:
variables should be declared with lowercase letters unless you are defining a class